tests/overlay_tests.cpp

```diff
@@ -570,22 +570,22 @@ TEST_F(OverlayTest, ROOT_checkMesosNetwork)
   ASSERT_SOME(json);
 
   Result<JSON::String> network = json->find<JSON::String>("name");
-  ASSERT_SOME(network);
+  ASSERT_SOME(network) ;
   EXPECT_EQ(network.get(), OVERLAY_NAME);
 
-  Result<JSON::Boolean> ipMasq = json->find<JSON::Boolean>("ipMasq");
-  ASSERT_SOME(ipMasq);
+  Result<JSON::Boolean> ipMasq = json->find<JSON::Boolean>("delegate.ipMasq");
+  ASSERT_SOME(ipMasq) << "CNI config: " << cniConfig.get();
   EXPECT_EQ(ipMasq.get(), false);
 
-  Result<JSON::Number> mtu = json->find<JSON::Number>("mtu");
-  ASSERT_SOME(mtu);
+  Result<JSON::Number> mtu = json->find<JSON::Number>("delegate.mtu");
+  ASSERT_SOME(mtu) << "CNI config: " << cniConfig.get();
   EXPECT_EQ(mtu.get(), 1420);
 
-  Result<JSON::Object> ipam = json->find<JSON::Object>("ipam");
-  ASSERT_SOME(ipam);
+  Result<JSON::Object> ipam = json->find<JSON::Object>("delegate.ipam");
```

Comment:
Can you add a link to where this config was changed?

Presumably these fields are now found in a deeper JSON object layer now
`"delegate" : { "ipMasq" : ... }`